FBTweak/FBTweakShakeWindow.m

```diff
@@ -30,6 +30,14 @@ - (instancetype)initWithFrame:(CGRect)frame {
     self->_shakeEnabled = YES;
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(_applicationWillResignActiveWithNotification:) name:UIApplicationWillResignActiveNotification object:nil];
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(_applicationDidBecomeActiveWithNotification:) name:UIApplicationDidBecomeActiveNotification object:nil];
+
+#if FB_TWEAK_ENABLED
```

Comment:
why it's required?

Reply:
take a look at the `_shouldPresent...` method. This window should not be used when tweaks are not enabled (release mode). We are extra careful, and even if by mistake the user adds this window it will not show.

Reply:
In that case I'd suggest you to consider the following approaches:

* Wrap `_presentTweaks` method in a macro and supply an empty implementation when you don't want *TWEAK*s to appear. This is superior to current approach since it disable the presentation method, which is called from multiple places. In this case, I guess, `_shouldPresent`'s logic can be simplified (or even removed).
* Introduce a compilation error when compiling for release and *TWEAK*s is built (IMO it's the better approach).